libs/k8s-client/tests/ApiClient/PodsApiClientFunctionalTest.php

```diff
@@ -73,7 +77,19 @@ public function testReadLog(): void
 
         $this->baseApiClient->create((string) getenv('K8S_NAMESPACE'), $pod);
         sleep(5);
-        $log = $this->apiClient->readLog('test-resource-1');
-        self::assertStringContainsString('Hello World', $log);
+        $logStream = $this->apiClient->readLog('test-resource-1');
+        self::assertInstanceOf(StreamInterface::class, $logStream);
+        self::assertSame(12, $logStream->getSize());
+        self::assertSame('Hello', $logStream->read(5));
+        self::assertSame(' World', $logStream->read(6));
+        $logStream->getContents();
+        self::assertTrue($logStream->eof());
+    }
```

Comment:
Jeste me napadlo zkusit otestovat to s `follow`, ze ten stream bude prubezne predavat novy logy, ale to by bylo celkem komplikovany na setup, takze to nechme byt.

Muselo by se:
* nahodit Pod, ktery bude postupne po nejakou dobu produkovat logy
* nahodit cteni logu
* checknout, ze to okamzite vrati existujici historii logu, ale neukonci se, ale vrati nejake dalsi radky, jakmile je Pod vyprodukuje

Nejak si nedovedu moc predstavit tohle v PHP nasetupovat a synchronizovat.

Reply:
Zní to hezky ale reálně si to taky moc nedovedu představit.